src/createServer.js

```diff
@@ -1,3 +1,48 @@
+/* eslint-disable max-len */
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+const http = require('http');
+
+const { CASES_LIST } = require('./constants/CASES_LIST');
+const { getErrorMessages } = require('./utils/getErrorMessages');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const [text, params] = req.url.split('?');
+
+    const originalText = text.slice(1);
+    const targetCase = new URLSearchParams(params).get('toCase');
+
+    if (!originalText || !targetCase || !CASES_LIST.includes(targetCase)) {
```

Comment:
You already have these checks in `getErrorMessages`, don't duplicate them, just check for `errorMessages.length !== 0`
```suggestion
```